src/client/components/OrdersSummary/OrdersSummary.stories.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import OrdersSummary from './OrdersSummary.component';
+
+export default {
+  title: 'Components / OrdersSummary',
+  component: OrdersSummary,
+  argTypes: {
+    orders: { control: '' },
+  },
+};
+
+const template = ({ orders }) => (
+    <OrdersSummary orders={orders} />
+  );
+
+export const OrdersDisplay = template.bind({});
+
+OrdersDisplay.args = {
+  orders: [
+    {
+      id: 1,
+      user_id: 1,
+      status: 'NEW',
+      created_at: "2022-02-28 21:28:04",
+      updated_at: "2022-02-28 21:28:04",
+    },
+    {
```

Comment:
For order summary you will be fetching first this api `api/orders?user=user_id` and the response you are going to receive is looking like this: 
![image](https://user-images.githubusercontent.com/81110602/157091570-d5f5cad2-3310-4468-ae14-375c4353b804.png)

And after this when the user clicks on an order we will show the order dispaly which will call the api `api//orders/{id}`  which returns something like what you have here.

Reply:
Thank you for Info Florentina.. Please have a look once and give feedback please ..